docs/workflow/trimming/feature-switches.md

```diff
@@ -0,0 +1,56 @@
+# Libraries Feature Switches
+
+Starting with .NET 5 there are several [feature-switches](https://github.com/dotnet/designs/blob/master/accepted/2020/feature-switch.md) available which
+can be used to control the size of the final binary. They are available in all
+configurations but their defaults might vary as any SDK can set the defaults differently.
+
+## Available Feature Switches
+
+| MSBuild property name | C# property name | Description |
+|-|-|-|
+| EnableUnsafeUTF7Encoding | System.Text.Encoding.EnableUnsafeUTF7Encoding |  Insecure UTF-7 encoding is trimmed when set to false |
+| EventSourceSupport | System.Diagnostics.Tracing.EventSource.IsSupported | Any EventSource related code or logic is trimmed when set to false |
+| InvariantGlobalization | System.Globalization.Invariant | All globalization specific code and data is trimmed when set to true |
+| UseSystemResourceKeys | System.Resources.UseSystemResourceKeys |  Any localizable resources for system assemblies is trimmed when set to true |
+| - | System.Diagnostics.Debugger.IsSupported | Any dependency that enables better debugging experience to be trimmed when set to false |
+| - | System.Net.Http.EnableActivityPropagation | Any dependency related to diagnostics support for System.Net.Http is trimmed when set to false |
+
+Any feature-switch which defines property can be set in csproj file or
+on the command line as any other MSBuild property. Those without predefined property name
+the value can be set with following XML tag in csproj file.
+
+```xml
+<RuntimeHostConfigurationOption Include="<csharp-property-name>"
+                                Value="false"
+                                Trim="true" />
+```
+
+## Adding New Feature Switch
+
+The primary goal of features switches is to produce smaller output by removing code which is
+unreachable under feature condition. The typical approach is to introduce static bool like
+property which is used to guard the dependencies which can be trimmed when the value is flipped.
+Ideally, the static property should be located in type which does not have any static initialization
```

Comment:
How do you initialize the property if you don't have any static initialization logic?

3 of the existing ones (EventSource, UseSystemResourceKeys, GlobalizationMode.Invariant) all have static initialization logic to initialize these properties.

Reply:
Tweaked the wording a bit